pmtiles/server.go

```diff
@@ -132,17 +120,23 @@ func (server *Server) Start() {
 							totalSize -= resp.size
 						}
 					}
-					cacheSize.Set(float64(len(cache)))
+					server.metrics.updateCacheStats(totalSize, len(cache))
 				}
 				key := req.key
 				if val, ok := cache[key]; ok {
 					evictList.MoveToFront(val)
 					req.value <- val.Value.(*response).value
+					server.metrics.cacheRequest(key.name, "hit")
 				} else if _, ok := inflight[key]; ok {
 					inflight[key] = append(inflight[key], req)
+					server.metrics.cacheRequest(key.name, "inflight")
```

Comment:
Does an extra "inflight" status make sense here? Or should those just be treated as a hit or a miss?

Reply:
Also, I could include root vs. leaf?

Reply:
I don't know if `inflight` is interesting on its own, since any spatially correlated client will generate N amounts of those, I think I'd treat them as a hit because they don't generate a network request on their own. 

What are our goals here? to optimize cache eviction?

Reply:
👍  I'll change that to a "hit"

The goal is just to help you tune the cache size: decide if it needs to be bigger, then after increasing how much impact did that make on hit rate/overall latency.